CoeursVaillants-2/CoeursVaillants-2.html

```diff
@@ -15,136 +22,140 @@
 				<input type="number" name="attr_cha" class='sheet-inpnum2cbig' style="margin-right: 10px; width: 45px" value='10' min="0" max="20" />
-                <input type="checkbox" class="check-button check-buttonComp" id="checkCompCha" value="1" name="attr_CompCha"/>
-                <label for="checkCompCha">Comp.</label>
+        <input type="checkbox" class="check-button check-buttonComp" id="checkCompCha" value="1" name="attr_CompCha"/>
+        <label for="checkCompCha">Comp.</label>
 			</div>
-			<div class="sheet-infoDesc" style="margin: -4px 0 9px 10px;">Charisme</div>
+			<div class="sheet-infoDesc sheet-marginCarac">Charisme</div>
+			<div class="sheet-libel1" style="margin: 10px 0px 0px 0px; padding: 0px;">Compensations</div>
+			<textarea name="attr_compensations" style='margin: 0px; width: 270px; height: 40px;'></textarea>
 		</div>
-        <!-- Partie droite -->
+    <!-- Partie droite -->
 		<div class="sheet-mycol" style="width: 66%;">
-            <!-- Ligne du haut -->
+    <!-- Ligne du haut -->
 			<div class="sheet-ligne sheet-borderBottom">
-                <!-- CADRE PV -->
+        <!-- CADRE PV -->
 				<div class="sheet-mycolbrd sheet-borderRight" style="width: 49%;">
 					<div class="sheet-ligne" style="text-align: center;">
 						<div class="sheet-libel3" style="width: 190px;">Points de Vie</div>
 					</div>
 					<div class="sheet-ligne" style="text-align: center; margin-bottom: 8px;">
 						<input type="number" name="attr_pv" class='sheet-inpnum2cbig' style="width: 53px; vertical-align: bottom;" value='0'>
-                        <div class="sheet-libel1" style="width:10px; vertical-align: bottom;">/</div>
-                        <input type="number" name="attr_pvmax" class='sheet-inpnum2c' style="width: 45px; margin-right: 20px; vertical-align: bottom" value='0' />
-                        <div class="sheet-mycol">
-                            <div class="sheet-infoDesc" style="display: block;">Temp.</div>
-                            <input type="number" name="attr_pvTemp" class='sheet-inpnum2c' style="width:40px;" value='0'/>
-                        </div>
-                        <div class="sheet-mycol">
-                            <div class="sheet-infoDesc" style="display: block;">Aggr.</div>
-                            <input type="number" name="attr_pvAggr" class='sheet-inpnum2c' style="width:40px;" value='0'/>
-                        </div>
+            <div class="sheet-libel1" style="width:10px; vertical-align: bottom;">/</div>
+            <input type="number" name="attr_pv_max" class='sheet-inpnum2c' style="width: 45px; margin-right: 20px; vertical-align: bottom" value='0' />
```

Comment:
changing attributes like this should come with [version handling](https://wiki.roll20.net/Character_Sheet_Development/Pattern_Libraries#Sheet_Versioning) to convert the data on existing sheets and prevent destruction of data for existing games

Reply:
Sure, I didn’t anticipate it. I’m looking into sheetworker stuff to add a migration script.